src/pe_reports/report_generator.py

```diff
@@ -22,21 +43,49 @@ def load_template():
 
 def export_set(prs):
     """Export PowerPoint report set to output directory."""
-    pptx_out = "Customer_ID_Posture_Exposure.pptx"
-    prs.save(os.path.join(SAVE_PATH, pptx_out))
+    try:
+        pptx_out = "Customer_ID_Posture_Exposure.pptx"
+        prs.save(os.path.join("/output", pptx_out))
+    except OSError:
+        print("No output available.")
     return
 
 
+def generate_reports(data, data_dir, out_dir):
+    """Gather assets to produce reports."""
+    # TODO: build code to connect customer db, encrypt and embed pdf reports.
+    # Issue #7: https://github.com/cisagov/pe-reports/issues/7
+
+
 def main():
+    """Set up logging and call the generate_reports function."""
+    args: Dict[str, str] = docopt.docopt(__doc__, version=__version__)
+
+    # Set up logging
+    logging.basicConfig(
+        format="%(asctime)-15s %(levelname)s %(message)s", level=logging.INFO
+    )
+
+    # TODO: Add generate_reports func to handle cmd line arguments and function.
+    # Issue #8: https://github.com/cisagov/pe-reports/issues/8
+    generate_reports(
+        args["REPORT_DATE"], args["DATA_DIRECTORY"], args["OUTPUT_DIRECTORY"]
+    )
+
     """Generate PDF reports."""
-    print("\n [Info] Loading Posture & Exposure Report Template, Version:", __version__)
+    logging.info(
+        f"[Info] Loading Posture & Exposure Report Template, Version :, {__version__}"
+    )
     prs = load_template()
 
-    print("\n [Info] Generating Graphs ")
+    logging.info("[Info] Generating Graphs ")
     Pages.cover(prs)
     Pages.overview(prs)
     export_set(prs)
 
+    # Stop logging and clean up
+    logging.shutdown()
+
 
 if __name__ == "__main__":
     sys.exit(main())
```

Comment:
You do not return anything from `main()`. This should only be included if the function has return values indicating normal behavior or errors.

```suggestion
    main()
```

Reply:
Now that `main()` returns a 0 or a 1, you should put this back the way it was:
```
sys.exit(main())
```

Reply:
Resolved in 7a0d456